src/lib/coda_base/user_command_payload.mli

```diff
@@ -25,7 +25,7 @@ module Body : sig
 
   module Stable : sig
```

Comment:
you can use `%%versioned:` here, and remove `bin_io` and `version` from the deriving list